osu.Game/Screens/Play/Player.cs

```diff
@@ -159,6 +160,8 @@ private void load(AudioManager audio, BeatmapDatabase beatmaps, OsuGameBase game
                 scoreOverlay,
                 pauseOverlay
             };
+
+            mouseWheelDisabled = config.GetBindable<bool>(OsuConfig.MouseDisableWheel);

```

Comment:
can you put this line beneath `dimLevel` above, and add a newline before the `try` after it?